Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -550,7 +550,7 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
         let longTaskEvent = RUMLongTaskEvent(
             dd: .init(
                 browserSdkVersion: nil,
-                discarded: nil,
+                discarded: false,
```

Comment:
No value in setting this one. If false won't have any effect and would only increase payload size.